test/test_nn.py

```diff
@@ -9721,6 +9721,48 @@ def _test_dropout(self, cls, device, input, memory_format=torch.contiguous_forma
         module.__repr__()
         str(module)
 
+    def _test_dropout_discontiguous(self, cls, device, memory_format=torch.contiguous_format):
+        # In this test, we verify that dropout preserves the layout and data for different memory formats.
```

Comment:
this test does not test that layout was preserved, so this is a misleading comment.